### PR TITLE
Set to nullptr after reset

### PR DESCRIPTION
**Pull request by sloriot**

### CODE REVIEW

File: STL_Extension/include/CGAL/Handle.h [hunk 1]
```diff
@@ -71,7 +71,10 @@ class Handle
       if (PTR)
       {
         if (--PTR->count==0)
+        {
           delete PTR;
+          PTR=nullptr;
+        }
```
Comment (lrineau) [inline STL_Extension/include/CGAL/Handle.h]: What is the gain?

**Comment by efifogel** [comment]
Just in case you invoke the destructor when the count is already zero.
Without it, the count will simply continue to decrease, which is also
harmless, but I think cleaner this way.
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

File: STL_Extension/include/CGAL/Handle.h (hunk 1 above)
Comment (lrineau) [reply STL_Extension/include/CGAL/Handle.h]: Actually, the next line, after the `if`, is already `PTR=0`.

Comment (sloriot) [reply STL_Extension/include/CGAL/Handle.h]: Right, I should have looked more closely, we can probably close the PR. @efifogel is it fine with you?

**Comment by efifogel** [comment]
Yep
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Pull request closed without merging**